src/tree_sortable.rs

```diff
@@ -51,18 +49,13 @@ impl FromGlib<i32> for SortColumn {
     }
 }
 
-pub trait TreeSortableExt {
-    // -- manual --
+pub trait TreeSortableExtManual {
     fn set_default_sort_func<F>(&self, sort_func: F)
         where F: Fn(&Self, &TreeIter, &TreeIter) -> Ordering + 'static;
     fn set_sort_func<F>(&self, sort_column_id: SortColumn, sort_func: F)
         where F: Fn(&Self, &TreeIter, &TreeIter) -> Ordering + 'static;
     fn get_sort_column_id(&self) -> Option<(SortColumn, SortType)>;
-    fn set_sort_column_id(&self, sort_column_id: SortColumn, order: SortType);
```

Comment:
This manual function was intended to replace the original one changing its signature, so the way to go here is to ignore the generated one instead of removing it here.